src/main/java/org/jenkinsci/plugins/docker/swarm/DockerSwarmAgentTemplate.java

```diff
@@ -57,7 +58,7 @@ public DockerSwarmAgentTemplate() {
 
     @DataBoundConstructor
     public DockerSwarmAgentTemplate(final String image, final String hostBinds, final String hostNamedPipes, final String dnsIps,
-            final String dnsSearchDomains, final String command, final String user, final String workingDir,
+            final String dnsSearchDomains, final String unixCommand,final String windowsCommand, final String user, final String workingDir,
```

Comment:
Note that this was an incompatible change. For example, a JCasC bundle that worked with 1.11 will not work with 1.12, and vice-versa. Recommend renaming `unixCommand` back to `command` before releasing this.

Reply:
Many thanks for the hint! I created a PR to rename the needed parts: https://github.com/jenkinsci/docker-swarm-plugin/pull/127
Did I rename enough or do I need to rename more `unixCommands` to `command`?